src/platform/web/ui/login/LoginView.js

```diff
@@ -76,3 +77,14 @@ class StartSSOLoginView extends TemplateView {
         );
     }
 }
+
+class StartOIDCLoginView extends TemplateView {
+    render(t, vm) {
+        return t.div({ className: "StartOIDCLoginView" },
+            t.a({
+                className: "StartOIDCLoginView_button button-action secondary",
+                href: vm => (vm.isBusy ? "#" : vm.authorizationEndpoint),
```

Comment:
As mentioned with the view model, I'd only calculate the redirect url once we've clicked this button/link. Once it's clicked, we should show a spinner so people don't think it didn't work. Once the redirect url is known, we programatically redirect.

Reply:
Done in 62054783